framework/src/engine/endpoint/system.ts

```diff
@@ -96,4 +97,14 @@ export class SystemEndpoint {
 			standardEvent: standardEventDataSchema,
 		};
 	}
+
+	public async getMetricsReport(ctx: RequestContext): Promise<unknown> {
+		if (!defaultMetrics.enabled()) {
+			throw new Error('metrics is not enabled');
+		}
+		if (ctx.params && ctx.params.inJSON === true) {
+			return defaultMetrics.report(ctx.params.inJSON);
+		}
+		return defaultMetrics.report();
```

Comment:
To make it consistent with other endpoints, the default format without any params could be JSON, while the string format required by Prometheus could be provided by setting a request param.